src/core/server/integration_tests/ci_checks/jest.integration.config.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+module.exports = {
+  bail: true, // only report 1 issue
+  // TODO replace the line below with
+  // preset: '@kbn/test/jest_integration_node
+  // to do so, we must fix all integration tests first
+  // see https://github.com/elastic/kibana/pull/130255/
+  preset: '@kbn/test/jest_integration',
```

Comment:
nit: since the number of tests is controlled here, we probably can apply this TODO.

WDYT?

Reply:
>  so open handles can cause the test to be running indefinitely 

Then we can't do here I'm afraid, there ARE open handles when starting and stopping the Kibana server quickly. 

Last one to date would probably be:

https://github.com/elastic/kibana/blob/cdaa3d3851514f646e23a7485302d096237b2803/x-pack/plugins/observability/server/services/slo/slo_installer.ts#L24-L32

Reply:
Fair point!